content/ops/managing-users.md

```diff
@@ -5,6 +5,12 @@ menu:
 title: Managing Users
 ---
 
+## Note for GovCloud environment
+
+Only single sign-on user accounts are allowed. Service accounts, such as deployer credentials, are permissible with the condition that they are scoped to a particular space with limited access.
+
+No local accounts to UAA shall be created for user access.
```

Comment:
I don't think our users will know what UAA or "local accounts" are, so we should probably just strike this line, or move it to documentation for operators.